packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -142,6 +149,18 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
   void _setupTabController() {
     _tabController?.dispose();
     _tabController = TabController(length: widget.tabs.length, vsync: this);
+
+    _handleScreenChange(widget.tabs[_tabController.index]);
+    _tabController.addListener(() {
```

Comment:
should we make sure this controller is disposed on disposal of this stateful widget?